src/pubsub/mod_pubsub_db_rdbms.erl

```diff
@@ -81,7 +81,6 @@
 %% ------------------------ Backend start/stop ------------------------
 
 init(HostType, Opts) ->
-    mod_pubsub_db_mnesia:init(HostType, Opts),
```

Comment:
Why is it safe to delete this?
Any idea why was the mnesia backend even started here?

Note that `mod_pubsub_db_mnesia:stop/2` is still called in line 254.

Reply:
It passed tests, so haven't investigated too deep. Not sure *why* it is there in the first place :)
Maybe there is some shared logic used by both backends. I thought there was at some point.
Could check again.

Reply:
At least that call would do nothing, if someone tries to use mod_pubsub without mnesia configured (i.e. all errors are just ignored inside).

Reply:
it is safe for start. I think, it was used when someone was adding rdbms backend, but was not cleaned up after rdbms backend was finished.

Reply:
I would take a look inside to ensure that the RDBMS backend is not reusing any logic of the mnesia one...